### PR TITLE
Correct dashboard figures from posting records

### PR DESCRIPTION
**Pull request by yareda**
HRD and stock status figures are correctly pulled from posting table.

### CODE REVIEW

File: app/models/posting_item.rb [hunk 1]
```diff
@@ -32,4 +32,9 @@
 
 class PostingItem < ApplicationRecord
     belongs_to :posting
+    belongs_to :commodity_category
+
+    def stock_balance
+    	PostingItem.sum(:quantity).where(account_id: 7).group(:commodity_category_id)
```
Comment (nathnael) [inline app/models/posting_item.rb]: The account_id shouldn't be hard coded. We can use something like this: "Account.find_by({'code': :stock})"

**Review by nathnael — CHANGES_REQUESTED** [review]

File: app/models/posting_item.rb (hunk 1 above)
Comment (yareda) [reply app/models/posting_item.rb]: I agree. Not supposed to be pushed in the PR. It was an experimentation code. Removed!

**Review by nathnael — APPROVED** [review]

File: app/models/operation.rb [hunk 2]
```diff
@@ -57,4 +57,37 @@ def reset_plan_id
       self.hrd_id = nil
     end
   end
+
+  def regional_request_in_operation
+    query = <<-SQL
```
Comment (nathnael) [inline app/models/operation.rb:62]: Here is an alternative way to get the same result using ActiveRecord:
RegionalRequest.joins(:operation).where(:operation_id => self.id).group(:region_id, :operation_id).count(:id)

**Review by nathnael — APPROVED** [review]

File: app/models/operation.rb (hunk 2 above)
Comment (yareda) [reply app/models/operation.rb:62]: Neat! I'll change it to AR then.

**Pull request merged**